### PR TITLE
jkfar4 halloween page

### PR DESCRIPTION
**Pull request by jkfar4**
Pull request practice

### CODE REVIEW

File: participants/julian-farinaccio/index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1> Spooky dance moves!</h1>
+    <div class="grid-container">
+        <img src="../julian-farinaccio/images/spooky-definitelynot.gif">
+        <img src="../julian-farinaccio/images/spooky-ghost.gif">
+        <img src="../julian-farinaccio/images/spooky-pumpkin.gif">
+        <img src="../julian-farinaccio/images/spooky-light.gif">
```
Comment (ericjiang97) [inline participants/julian-farinaccio/index.html]: Woah! You might want to a more direct reference here, something like

``

Comment (jkfar4) [reply participants/julian-farinaccio/index.html]: Right! They are now updated

**Review by ericjiang97 — COMMENTED** [review]
Also don't forget to add your page onto the main site!

**Review by josh- — APPROVED** [review]
So spooky 👻🎃 Thanks for the PR!

**Pull request merged**